src/UI/templates/default/Button/button.less

```diff
@@ -0,0 +1,25 @@
+.btn-tag {
+	&.btn-tag-inactive {
+		cursor: default !important;
```

Comment:
Without hovering there is no visiual difference between clickable and non-clickable tags, right? I think this is worth mentioning, but I also think it is ok. I think it is a good idea that tags are a type of button.